src/picongpu/submit/hermes-metacentrum-cz/doom_profile.tpl

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Copyright 2013-2015 Axel Huebl, Anton Helm, 
+#                     Rene Widera, Vojtěch Horný
```

Comment:
@Horymir001 if you don't mind, pls remove them. we had some problems with non-ascii chars in René's `é` before :(